web/src/css_variables.ts

```diff
@@ -37,6 +37,7 @@ export const media_breakpoints = {
     cb4_min: cb4 + "px",
     cb5_min: cb5 + "px",
     short_navbar_cutoff_height: "600px",
+    modal_sidebar_collapse_breakpoint: 1000 / 14 + "em",
```

Comment:
Maybe `settings_overlay_sidebar_collapse_breakpoint` would be more clear here.

Reply:
Sure!

Reply:
Actually, the reason I did it more broadly phrased is that it's also used for the subscriptions modal. Those modals share a lot of CSS.

Reply:
We can name it as `overlay_sidebar_collapse_breakpoint` also rather than using modal. Or we also refer to subscriptions UI as stream settings or channel settings as well, so having `settings` should also be fine.